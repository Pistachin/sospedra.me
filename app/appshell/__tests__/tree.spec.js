/* global describe, expect, it */
import React from 'react'
import { shallow } from 'enzyme'

import Appshell from '../index'

describe('Appshell wrapper suite', () => {
  const wrapper = shallow(<Appshell>
    <div>A</div>
    <div>B</div>
  </Appshell>)

  it('should have as many children as sent by props', () => {
    expect(wrapper.children().length).toBe(2)
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
