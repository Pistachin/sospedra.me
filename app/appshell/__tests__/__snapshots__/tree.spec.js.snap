exports[`Appshell wrapper suite should match the snapshot 1`] = `
<div
  className="appshell_1o99bj1"
  id="appshell">
  <div>
    A
  </div>
  <div>
    B
  </div>
</div>
`;
